src/Microsoft.AspNet.Mvc.Core/Logging/ScopeValues.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Framework.Internal;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Mvc.Logging
+{
+    public class ScopeValues : LoggerStructureBase
```

Comment:
not a goal, this is too generic. Try what I'm suggesting and then look at the results in ELK